src/docs/guides/arc_WorkingWithBranches.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+title: Working with branches
+date: 2024-08-17
+author:
+- name: Viktoria Petrova
+  github: toriapetrova
+add toc: true
+add support: true
+add sidebar: _sidebars/mainSidebar.md
+---
+
+
+<!-- TODO article about licensing  -->
+
+## Git branches
+
+Branches in Git allow users to work on and develop new features of their projects without affecting or changing other branches in the repository. 
+
+Imagine you're writing a book. The main path you're working on is the main storyline, which we'll call the main branch. But at some point, you get a new idea for an alternative storyline. Instead of changing the main storyline right away, you create a new path (or branch). Now, you can write and develop this new idea without affecting the main storyline. If you like how the alternative idea turns out, you can merge it back into the main storyline. If not, you can simply discard it, and the main storyline remains untouched.
```

Comment:
This is repetitive of the section written in the Fundamentals article on version control. 

Just reference the section here